CommunicationLayer/CommDeviceControl/RadioConnectionService.cpp

```diff
@@ -30,18 +31,25 @@ bool RadioConnectionService::connectToDataSource()
 {
    if (serialPort_.open(QIODevice::ReadWrite))
    {
+      connect(&serialPort_, SIGNAL(readyRead()),
+         this, SLOT(handleSerialDataIncoming()), Qt::UniqueConnection);
       emit connectionSucceeded();
       return true;
    }
    else
    {
+      qDebug() << serialPort_.errorString();
       emit connectionFailed(serialPort_.errorString());
       return false;
    }
 }
 
-void RadioConnectionService::disconnectFromDataSource()
+void RadioConnectionService::handleSerialDataIncoming()
```

Comment:
Addressed in c6ab365004b362711b000b44dac2295c734d4dac. Renamed RadioConnectionService to be RadioCommDevice.